static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -641,6 +641,7 @@ class RuleConditionsForm extends PureComponent<Props, State> {
                         searchSource="alert_builder"
                         defaultQuery={initialData?.query ?? ''}
                         {...getSupportedAndOmittedTags(dataset, organization)}
+                        metricAlert
```

Comment:
instead of adding a new prop can we add the supported prop to getSupportedAndOmittedTags which would help with it not showing up when the dataset is DDM etc